Makefile

```diff
@@ -17,11 +17,11 @@ watcher_SRC := $(wildcard $(CURDIR)/watcher/*.sh)
 
 SRC := $(common_SRC) $(koji_SRC) $(release_SRC) $(watcher_SRC)
 
-release_STEPS := $(patsubst %.sh,%,$(release_SRC))
+release_STEPS := $(patsubst %.sh,%,$(notdir $(release_SRC)))
 
 all:
 	@echo "use 'make release' to run all steps'"
-	@echo "use 'make STEP' to run individual steps: ${STEPS}"
+	@echo "use 'make STEP' to run individual steps: ${release_STEPS}"
```

Comment:
:+1:

Reply:
Actually, and I'm not sure it is still possible to just call a given step. Well, not before previously execute 'make release' at least one. Maybe this message should be removed?

Reply:
I think we can keep it. At least for me, when I started this tool, I tried the steps one by one :)

Reply:
but that implies that, from the first usage, the user needs to know about var_save and var_load that I like implicit inputs for each steps.

Reply:
Yes, the vars generated on the previous steps would be reused by the latter steps. Anyway, I think keeping this option is good for debug but useless for general release. It is your call :)